kong/plugins/jwt/schema.lua

```diff
@@ -16,5 +16,6 @@ return {
     secret_is_base64 = {type = "boolean", default = false},
     claims_to_verify = {type = "array", enum = {"exp", "nbf"}},
     anonymous = {type = "string", default = "", func = check_user},
+    cookie_name = {type = "string", default = "jwt_token"},
```

Comment:
We will need a migration for existing plugin definitions to insert this default value, otherwise you will run into a thread abort when trying to concat this to search `ngx.var`.

Reply:
@p0pr0ck5 So is there any way to do migration for this ? or I need to reset the configuration ? I have done below steps - 
1. `sudo kong migrations reset -c /etc/kong/kong.conf` This operation is irreversible, so I need to setup kong again.
Is there any way to to insert default value instead of resetting everything ?

Reply:
@sanketgandhi what @p0pr0ck5 meant is that this code requires migration code to be added to this PR.